### DISCUSSION TITLE
Output console log format

### DISCUSSION THREAD

**Discussion by boxedcode** [prisma/prisma#22929] [Q&A] [answered]
### Question

Hi!

Is there a way to customise the output format of logs generated by prisma?

For example, with `nginx` access logs, I can do something like this within an `http` block

```
    log_format compression '$remote_addr - $remote_user [$time_local] '
                           '"$request" $status $body_bytes_sent '
                           '"$http_referer" "$http_user_agent" "$gzip_ratio"';
```

Thanks.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @boxedcode 👋 

Thank you for raising this question.

Prisma Client does not directly support custom log formats. However, there is an open [issue](https://github.com/prisma/prisma/issues/10475) discussing the possibility of a custom logger interface. This would allow you to capture Prisma's log events and handle them with your own logger, potentially allowing you to customize the output format. You can add your use case and leave a 👍 on the issue to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by boxedcode** [comment]
Hi @ludralph , thanks for your response and yes, this does answer my question.
